sympy/solvers/tests/test_ode.py

```diff
@@ -3414,3 +3414,25 @@ def test_issue_15913():
 def test_issue_16146():
     raises(ValueError, lambda: dsolve([f(x).diff(x), g(x).diff(x)], [f(x), g(x), h(x)]))
     raises(ValueError, lambda: dsolve([f(x).diff(x), g(x).diff(x)], [f(x)]))
+
+def test_Airy_equation():
+    C1, C2 = symbols("C1 C2")
+    from sympy.functions import airyai, airybi
+    eq = f(x).diff(x, 2) - x*f(x)
+    assert classify_ode(eq) == ("2nd_linear_airy",'2nd_power_series_ordinary')
+    assert dsolve(eq) == Eq(f(x), C1*airyai(x) + C2*airybi(x))
+    eq = f(x).diff(x, 2) + 2*x*f(x)
+    assert classify_ode(eq) == ("2nd_linear_airy",'2nd_power_series_ordinary')
+    assert str(dsolve(eq)) == "Eq(f(x), C1*airyai(-2**(1/3)*x) + C2*airybi(-2**(1/3)*x))"
+
+def test_2nd_linear_bessel_equation():
+    C1, C2 = symbols("C1 C2")
+    from sympy.functions import besselj, bessely
+    eq = x**2*(f(x).diff(x, 2)) + x*(f(x).diff(x)) + (x**2 - 4)*f(x)
+    assert dsolve(eq) == Eq(f(x), C1*besselj(2, x) + C2*bessely(2, x))
+    eq = x**2*(f(x).diff(x, 2)) + x*(f(x).diff(x)) + (x**2 +25)*f(x)
+    assert classify_ode(eq) == ('2nd_linear_bessel', '2nd_power_series_regular')
+    eq = x**2*(f(x).diff(x, 2)) + x*(f(x).diff(x)) + (x**2)*f(x)
+    assert dsolve(eq) == Eq(f(x), C1*besselj(0, x) + C2*bessely(0, x))
+    eq = x**2*(f(x).diff(x, 2)) + x*(f(x).diff(x)) + (81*x**2 -S(1)/9)*f(x)
+    assert dsolve(eq) == Eq(f(x), C1*besselj(1/3, 9*x) + C2*bessely(1/3, 9*x))
```

Comment:
Perhaps, you should Simpify these terms. Eg. `S(1)/3`.
Python2.7 sometimes gives an error if not simpified